plugins/kotlin/idea/resources-en/inspectionDescriptions/RedundantRequireNotNullCall.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <body>
-<p>Reports redundant <code>requireNotNull</code> or <code>checkNotNull</code> call on not-nullable expressions</p>
+<p>Reports <code>requireNotNull</code> or <code>checkNotNull</code> call on non-nullable expressions.</p>
```

Comment:
Do we really want to remove "redundant" word here? For it makes sense

Reply:
My reason was that **all** of such checks are redundant on non-nullable expressions, so the specification isn't necessary. In UI, the shorter the better.
Although if this isn't tru, let's return `redundant`.